Warning for material property animation (#769)
anatawa12/AvatarOptimizer: CHANGELOG-PRERELEASE.md

    - Converts Entry / Exit to 1D BlendTree `#854` `#867`
    - Merges multiple Direct BlendTree to single Direct BlendTree `#870`
    - Removes meaningless Animator Layers `#870`
- Warning for material animation in Merge Skinned Mesh `#769`

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Author: ここに関しては「最初から今までずっと」なのでinではないかなと思います

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Reviewer: あー、今はfirst versionではないのか
じゃあ、first versionから今まで、という表記にしたいかもです

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Author: 0.0.1からあるしremoveEmptyRendererObjectの追加等が行われてるのでfirst versionではないと思います。

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Reviewer: んー…どう変えればよいだろう…

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Reviewer: ```suggestion
  - Until this version, Merge Skinned Mesh does not support animating material properties differently. (In other words, it can be broken.)
```
こうかな

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Author: Until this versionだと今まで対応してない (= 今は対応してる)になっちゃうけど、昔も今もある程度の将来も対応しないので、`Until this version, Merge Skinned Mesh does not support animating material properties differently.`も不適切だと思います

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Reviewer: > 昔も今もある程度の将来も対応しない

そうなんだ
じゃあ…SinceもUntilも要らないのでは?
```suggestion
  - Merge Skinned Mesh does not support animating material properties differently. (In other words, it can be broken.)
```

>   - Since first version of Merge Skinned Mesh, animating material properties differently is not supported (in other words, it might be broken).

Author: 一番最初から対応してなかったんだよと明記するつもりで入れてたけどまぁなくていいか
